pydrive2/files.py

```diff
@@ -220,7 +221,10 @@ def GetContentString(
             self.FetchContent(mimetype, remove_bom)
         return self.content.getvalue().decode(encoding)
 
-    def GetContentFile(self, filename, mimetype=None, remove_bom=False):
+    @LoadMetadata
```

Comment:
we want to avoid extra API call, if we can't do this in this call we will need to create a separate function that does only one simple thing w/o extra calls